scripts/js.js

```diff
@@ -57,18 +57,35 @@ function setCursor() {
     var scaleX = canvas.getBoundingClientRect().width / canvas.width;
     cursorCanvas.width = brushSize * 2 * scaleX;
     cursorCanvas.height = brushSize * 2 * scaleX;
+
+    // two cursors – for visibility against dark backgrounds
     var cursorCtx = cursorCanvas.getContext('2d');
+    var outlineCtx = cursorCanvas.getContext('2d');
 
+    cursorCtx.strokeStyle = '#000000';
     cursorCtx.beginPath();
     cursorCtx.arc(
         cursorCanvas.width / 2,
         cursorCanvas.height / 2,
-        brushSize * scaleX,
+        brushSize * scaleX - 2,
```

Comment:
Why do -2 and -1, instead of -1 and 0 or 0 and +1?

Reply:
I played around with both, but I was noticing clipping issues for the circles for -1 / 0 and 0 / +1. Making them a tiny bit smaller kept the circles intact in my tests.